File: first/src/test/java/com/pauldaniv/template/lib/first/util/TestUtilTest.java
package com.pauldaniv.template.lib.first.util;

import static org.assertj.core.api.Assertions.assertThat;

import org.junit.Test;

public class TestUtilTest {
    @Test
    public void join() {
        assertThat(TestUtil.join("one", "two")).isEqualTo("onetwo");
    }
}

File: first/src/test/kotlin/com/pauldaniv/template/lib/first/util/TestUtilTest.kt
package com.pauldaniv.template.lib.first.util

import org.assertj.core.api.Assertions.assertThat
import org.junit.jupiter.api.Test

class TestUtilTest {
    //todo add loging
  @Test
  fun join() {
    assertThat(TestUtil.join("one", "two")).isEqualTo("onetwo")
  }
}
